go/http-net/http-net.go

```diff
@@ -0,0 +1,36 @@
+package httpnet
+
+import (
+	"context"
+	"net/http"
+
+	"google.com/sqlcommenter/core"
+)
+
+type HttpNet struct {
```

Comment:
HTTPRequestTagger ?

Reply:
Do I need to rename?

Reply:
Yes, the interface is just `RequestTagger` as it can implement both HTTP and GRPC or any other framework level tagger.

And this one here is for http library. So, HTTPRequestTagger should be good.

Reply:
Also, this Http is part of golang core module. So, only HTTPRequestTagger should be fine i think.